File: Java/Strings/palindrome.java
public class palindrome {
    public static void main(String[] args) {
        palindrome("madam");
    }
    public static void palindrome(String s){
        String rev = "";
        for(int i=s.length()-1;i>=0;--i)
            rev+=s.charAt(i);
        
        if(s.equalsIgnoreCase(rev))
            System.out.println("Entered string is palindrome");
        else
            System.out.println("Entered string is not palindrome");
    }
}

File: Python/Strings/palindrome.py
def palindrome(s):
    rev = ""
    for i in range(len(s)-1,-1,-1):
        rev+=s[i]
    if rev == s:
        print("Entered string is palindrome")
    else:
        print(("Entered string is not palindrome"))
        
palindrome('madam')